NoSQL/README.md

```diff
@@ -36,14 +36,14 @@ Start up KVLite in a Docker container. You must give it a name. Startup of
 KVLite is the default `CMD` of the Docker image:
 
 ```shell
-docker run -d --name=kvlite oracle/nosql:19.5-ce
+docker run --name=kvlite --env KV_PROXY_PORT=8080 -p 8080:8080 oracle/nosql:20.3
```

Comment:
Why are you removing the `-d` parameter? That's what allows the container to run as a daemon, i.e. in the background.

Also, please remove the word "Docker" from the preceeding sentence.